x-pack/plugins/fleet/cypress/e2e/previleges_integrations_automatic_import.cy.ts

```diff
@@ -0,0 +1,159 @@
+/*
```

Comment:
nit: should that file be named `privileges` instead of `previleges`

Reply:
[757fb9b](https://github.com/elastic/kibana/pull/194948/commits/757fb9b2e6493a16036ae512bc84298fc8b8b400) fixes this